plugins/olm/commands/upgrade.md

```
@@ -0,0 +1,349 @@
+---
+description: Update an operator to the latest version or switch channels
+argument-hint: <operator-name> [namespace] [--channel=<channel>] [--approve]
+---
+
+## Name
+olm:upgrade
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)